### ISSUE TITLE
vagrant destroy fails after manually deleting linode

### ISSUE THREAD

**Issue by spacez320** [displague/vagrant-linode#56]
This causes your Vagrant build cycle to become stuck because it thinks the box is active when it actually no longer exists. Recreating a Linode box by hand with the same label does not address the problem.

To recreate:

1. create Linode machine with vagrant-linode
2. remove the produced machine through the Linode dash
3. run `vagrant destroy`

You should see this:

```
/home/matthew/.vagrant.d/gems/gems/linodeapi-0.2.2/lib/linodeapi/raw.rb:85:in `parse': API Error on linode.delete: [{"ERRORCODE"=>5, "ERRORMESSAGE"=>"Object not found"}] (RuntimeError)
	from /home/matthew/.vagrant.d/gems/gems/linodeapi-0.2.2/lib/linodeapi/raw.rb:79:in `error_check'
	from /home/matthew/.vagrant.d/gems/gems/linodeapi-0.2.2/lib/linodeapi/raw.rb:71:in `call'
	from /home/matthew/.vagrant.d/gems/gems/linodeapi-0.2.2/lib/linodeapi/raw.rb:62:in `block in make_call'
	from /home/matthew/.vagrant.d/gems/gems/linodeapi-0.2.2/lib/linodeapi/raw.rb:63:in `make_call'
	from /home/matthew/.vagrant.d/gems/gems/linodeapi-0.2.2/lib/linodeapi/raw.rb:44:in `method_missing'
	from /home/matthew/.vagrant.d/gems/gems/vagrant-linode-0.2.3/lib/vagrant-linode/actions/destroy.rb:16:in `call'
	from /usr/share/vagrant/lib/vagrant/action/warden.rb:34:in `call'
	from /home/matthew/.vagrant.d/gems/gems/vagrant-linode-0.2.3/lib/vagrant-linode/actions/connect_linode.rb:31:in `call'
	from /usr/share/vagrant/lib/vagrant/action/warden.rb:34:in `call'
	from /usr/share/vagrant/lib/vagrant/action/warden.rb:95:in `block in finalize_action'
	from /usr/share/vagrant/lib/vagrant/action/warden.rb:34:in `call'
	from /usr/share/vagrant/lib/vagrant/action/warden.rb:34:in `call'
	from /usr/share/vagrant/lib/vagrant/action/builder.rb:116:in `call'
	from /usr/share/vagrant/lib/vagrant/action/runner.rb:66:in `block in run'
	from /usr/share/vagrant/lib/vagrant/util/busy.rb:19:in `busy'
	from /usr/share/vagrant/lib/vagrant/action/runner.rb:66:in `run'
	from /usr/share/vagrant/lib/vagrant/action/builtin/call.rb:53:in `call'
	from /usr/share/vagrant/lib/vagrant/action/warden.rb:34:in `call'
	from /usr/share/vagrant/lib/vagrant/action/warden.rb:95:in `block in finalize_action'
	from /usr/share/vagrant/lib/vagrant/action/warden.rb:34:in `call'
	from /usr/share/vagrant/lib/vagrant/action/warden.rb:34:in `call'
	from /usr/share/vagrant/lib/vagrant/action/builder.rb:116:in `call'
	from /usr/share/vagrant/lib/vagrant/action/runner.rb:66:in `block in run'
	from /usr/share/vagrant/lib/vagrant/util/busy.rb:19:in `busy'
	from /usr/share/vagrant/lib/vagrant/action/runner.rb:66:in `run'
	from /usr/share/vagrant/lib/vagrant/action/builtin/call.rb:53:in `call'
	from /usr/share/vagrant/lib/vagrant/action/warden.rb:34:in `call'
	from /usr/share/vagrant/lib/vagrant/action/builtin/config_validate.rb:25:in `call'
	from /usr/share/vagrant/lib/vagrant/action/warden.rb:34:in `call'
	from /usr/share/vagrant/lib/vagrant/action/builder.rb:116:in `call'
	from /usr/share/vagrant/lib/vagrant/action/runner.rb:66:in `block in run'
	from /usr/share/vagrant/lib/vagrant/util/busy.rb:19:in `busy'
	from /usr/share/vagrant/lib/vagrant/action/runner.rb:66:in `run'
	from /usr/share/vagrant/lib/vagrant/machine.rb:214:in `action_raw'
	from /usr/share/vagrant/lib/vagrant/machine.rb:191:in `block in action'
	from /usr/share/vagrant/lib/vagrant/environment.rb:516:in `lock'
	from /usr/share/vagrant/lib/vagrant/machine.rb:178:in `call'
	from /usr/share/vagrant/lib/vagrant/machine.rb:178:in `action'
	from /usr/share/vagrant/plugins/commands/destroy/command.rb:31:in `block in execute'
	from /usr/share/vagrant/lib/vagrant/plugin/v2/command.rb:226:in `block in with_target_vms'
	from /usr/share/vagrant/lib/vagrant/plugin/v2/command.rb:220:in `each'
	from /usr/share/vagrant/lib/vagrant/plugin/v2/command.rb:220:in `with_target_vms'
	from /usr/share/vagrant/plugins/commands/destroy/command.rb:30:in `execute'
	from /usr/share/vagrant/lib/vagrant/cli.rb:42:in `execute'
	from /usr/share/vagrant/lib/vagrant/environment.rb:301:in `cli'
	from /usr/share/vagrant/bin/vagrant:174:in `<main>'
```

**Comment by displague** [comment]
Thanks. I'll look into this to confirm and correct if needed.

**Comment by spacez320** [comment]
For anyone having a similar issue -- I was able to recover the situation by destroying the `.vagrant` directory local to the project. That seemed to make Vagrant forget about the preexisting Linode build.

**Comment by displague** [comment]
I believe this has been addressed in v0.2.5.
